src/kernels/jupyter/session/jupyterSession.ts

```diff
@@ -196,27 +196,27 @@ export class JupyterSessionWrapper
 }
 
 export function getRemoteSessionOptions(
-    remoteConnection: IJupyterConnection,
-    resource?: Uri
+    _remoteConnection: IJupyterConnection,
```

Comment:
Leaving this as, this code is crucial for some other work
Comments will help figure out whats possible instead of leaving this in an issue or scm history